ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug report
+about: Create a report to file an error or unexpected behavior
+title: ''
+labels: bug, triage
+assignees: ''
+
+---
+
+<!-- Provide a general summary of the issue in the Title above -->
+
+## Current Behavior
+<!-- Tell us what happens instead of the expected behavior -->
+
+## Expected Behavior
+<!-- Tell us what should happen -->
+
+## Possible Solution
+<!-- Not obligatory, but suggest a fix/reason for the bug -->
+
+## Steps to Reproduce
+<!-- Provide a link to a live example, or an unambiguous set of steps
+     to reproduce this bug. -->
+
+1.
+
+## Context
+<!-- How has this issue affected you? What are you trying to accomplish?
+     Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Regression
+<!-- Is this issue a regression? (Yes / No)
+     If Yes, optionally please include version or commit id or PR# that caused this regression,
+     if you have these details -->
+
+## Your Environment
+<!-- Include as many relevant details about the environment you experienced the bug in -->
+* Version used:
+* Server setup and configuration:
+* Operating System and version (`uname -a`):
+
+## Don't forget to add labels!
+- component label (see project-specific labels)
+- `goodfirstissue`, `helpwanted` if needed
```

Comment:
BTW, "good first issue" is the only one written with spaces, while we don't have `help wanted`.